int32 id
geometry_msgs/Point point
std_msgs/String state
std_msgs/String label

